### ISSUE TITLE
Plugin does not work! please fix asap. 

### ISSUE THREAD

**Issue by IevaPuo** [drabiger/estimationsChromePlugin#7]

**Comment by rempel** [comment]
The correct task board class is now `.planTaskBoardPage`.

```
document.querySelectorAll(".PlannerRoot .planTaskBoardPage  .taskBoardColumn")
```

**Comment by drabiger** [comment]
Thanks for the hint. Your issue title is a bit rude, though.

**Comment by drabiger** [comment]
Will be fixed with version 1.65 of the plugin. Fasted way to get the version is to uninstall the extension, then install it again.

**Comment by IevaPuo** [comment]
Thanks! it works. I apologize for being rude, it happened again last week. I understand that it connects to chrome updates. 
So far we are using this plugin for production planning so it is complicated and frustrating when it does not work. Thanks again.
